src/components/ContactForm.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
+import { FormGroup, FormWrapper } from "./products/style";
+import Button from "./Button";
+
+const ContactForm = () => {
+  const [formData, setFormData] = useState({
+    fullName: "",
+    subject: "",
+    email: "",
+    body: "",
+  });
+
+  const [errors, setErrors] = useState({});
+
+  const handleChange = (e) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    const validationErrors = validateForm();
+
+    if (Object.keys(validationErrors).length === 0) {
+      console.log("Form submitted:", formData);
+      window.location.href = "/ContactSuccess";
+    } else {
+      setErrors(validationErrors);
+    }
+  };
+
+  const validateForm = () => {
+    const errors = {};
+
+    if (formData.fullName.length < 3) {
```

Comment:
Typically this would be done with a form validation library to ensure that it's robust - but this is fine for the CA